magda-correspondence-api/src/CEmailTplRender.ts

```diff
@@ -0,0 +1,82 @@
+import { Attachment } from "./SMTPMailer";
+import CContentApiDirMapper from "./CContentApiDirMapper";
```

Comment:
What does the first C stand for in CContent* and CEmail*? Just a personal preference but would rather prefer to see descriptive names for things instead of short forms.

Reply:
Ah~ I think I am a little bit old school on this~ 😄
`C` indicates it's a class that requires initialization.
If you think it's confusing, how about I just remove the `C`?

Reply:
Hmmm... never seen it before.
Probably remove. If it starts with a capital letter, it should be a class.

Reply:
Removed C 👍